_rules/table-header-cells-refer-to-data-cells-d0f69e.md

```diff
@@ -0,0 +1,324 @@
+- [Forming relationships between data cells and header cells][assigned]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The column header element has an [assigned][] cell, within the same `table` element.
+
+```html
+<table>
+	<tr>
+		<th>Time</th>
+	</tr>
+	<tr>
+		<td>05:41</td>
+	</tr>
+</table>
+```
+
+#### Passed Example 2
+
+Each column header element has assigned cells, within the same `table` element.
+
+```html
+<div role="table">
+	<div role="rowgroup">
+		<div role="row">
+			<span role="columnheader">Month</span>
+			<span role="columnheader">Top Temperature</span>
+		</div>
+	</div>
+	<div role="rowgroup">
+		<div role="row">
+			<span role="cell">July</span>
+			<span role="cell">40 C</span>
+		</div>
+		<div role="row">
+			<span role="cell">August</span>
+			<span role="cell">45 C</span>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed Example 3
+
+Each column header element has assigned cells within the same table element. In this example the column headers have cells that span multiple columns.
+
+```html
+<table>
+	<thead>
+		<tr>
+			<th>Projects</th>
+			<th>Exams</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td colspan="2">15%</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Passed Example 4
+
+Each row and column header element has assigned cells, within the same element having a [semantic role][] of `grid`.
+
+```html
+<table role="grid">
+	<thead>
+		<tr role="row">
+			<td></td>
+			<th role="columnheader">Breakfast</th>
+			<th role="columnheader">Lunch</th>
+			<th role="columnheader">Dinner</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr role="row">
+			<th scope="row" role="rowheader">Day 1</th>
+			<td>8:00</td>
+			<td>13:00</td>
+			<td>18:00</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Passed Example 5
+
+Each column header is assigned to a cell. Usage of `headers` attribute changes the relationship between column headers and cells.
+
+```html
+<table>
+	<tr>
+		<th id="col1">Column 1</th>
+		<th id="col2">Column 2</th>
+	</tr>
+	<tr>
+		<td></td>
+	</tr>
+	<tr>
+		<td headers="col2"></td>
+	</tr>
+</table>
+```
+
+### Failed
+
+#### Failed Example 1
+
+The column header ("Column 2"), does not have an assigned cell within the same `table` element.
+
+```html
+<table>
+	<thead>
+		<tr>
+			<th>Column 1</th>
+			<th>Column 2</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td>15%</td>
+		</tr>
+	</tbody>
+</table>
+```
+
+#### Failed Example 2
+
+The column header ('Column2') does not have an assigned cell within the same `table` element. In this case the usage of `headers` attribute removes cell association to the column.
+
+```html
+<table>
+	<tr>
+		<th id="col1"></th>
+		<th id="Column2"></th>
+	</tr>
+	<tr>
+		<td></td>
+		<td headers="col1"></td>
+	</tr>
+</table>
+```
+
+#### Failed Example 3
+
+The column header ('Column B') does not have an assigned cell within the same `table` element.
```

Comment:
This example does not have any column header corresponding to the definition "Column B".
If you don't want to use my suggestion (https://github.com/act-rules/act-rules.github.io/pull/979/files#r363636645) and prefer dismissing it, you still need to somehow have the description actually matching the example.